webpack.config.js

```diff
@@ -1,29 +1,14 @@
-var webpack = require('webpack');
+var getConfig = require('hjs-webpack')
+var webpack = require('webpack')
```

Comment:
Missing semicolon.<br>'require' is not defined.<br>'webpack' is defined but never used.